src/main/java/ExecutableDocuments.java

```diff
@@ -21,9 +36,15 @@ private void run(String[] args) {
         if (args.length > ARGS_IS_EMPTY && programArgumentsController.readProgramArguments(args) != USER_DO_NOT_USE_CONSOLE) {
             userOptions.setStrategy(programArgumentsController.readProgramArguments(args));
         } else {
-            userOptions.setStrategy(userInterfaceController.getUserDecision());
+            userOptions.setStrategy(userInterfaceController.getUserDecision(getNumberFromUser));
         }
         if (userOptions.getStrategy() == USER_OPTION_IS_COPY_FILE) fileStrategy = new CopyFile();
-        if (userOptions.getStrategy() == USER_OPTION_IS_READ_FILE) fileStrategy = new ReadFile();
+        else if (userOptions.getStrategy() == USER_OPTION_IS_READ_FILE) fileStrategy = new ReadFile();
+        // for test
+        if (fileStrategy.perform(userInterfaceController.getListOfPathFromUser(keyboardReader),userInterfaceController.getPathFromUser(keyboardReader))) {
+            System.out.println("Copied");
```

Comment:
zła warstwa ui to ui

Reply:
Dodałem userInterface.getMessage(). Jednak nie mam wytycznych co powonieniem wyświetlić po skopiowaniu plików.